.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishHolidayExtractorConfiguration.cs

```diff
@@ -11,13 +11,13 @@ public class EnglishHolidayExtractorConfiguration : BaseDateTimeOptionsConfigura
             new Regex(DateTimeDefinitions.YearRegex, RegexFlags);
 
         public static readonly Regex H1 =
-            new Regex(DateTimeDefinitions.HolidayRegex1, RegexFlags);
+            new Regex(DateTimeDefinitions.HolidayRegex1CS, RegexFlags);
```

Comment:
I guess these should be combined at the resources level. Do you see a negative side-effect of matching only once? Or a benefit of keeping multiple regexes?

Reply:
no I do not see negative side-effects, I have updated the file to use a single combined regex.